ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -135,7 +135,9 @@ CONTROL_PLANE_STACK_NAME=${CLUSTER_NAME}-control-plane
 COMPUTE_STACK_NAME_PREFIX=${CLUSTER_NAME}-compute
 
 # Create s3 bucket for bootstrap and proxy ignition configs
-aws s3 mb s3://"${INFRA_STACK_NAME}"
+S3_BUCKET_URI="s3://${INFRA_STACK_NAME}"
+aws s3 mb "${S3_BUCKET_URI}"
+echo ${S3_BUCKET_URI} > ${SHARED_DIR}/s3_bucket_uri
```

Comment:
Maybe we can directly delete the s3 bucket in this upi-install script on `trap` lines

Reply:
Of course, this way should also work. On your call.